src/SPIN/fix_nve_spin.cpp

```diff
@@ -296,6 +305,13 @@ void FixNVESpin::initial_integrate(int /*vflag*/)
     }
   }
 
+  // update fm_kspace if long-range
+  // remove short-range comp. of fm_kspace
+
+  if (long_spin_flag) {
+
```

Comment:
This if statement is empty. Either something needs to be added inside the curly braces or the whole block can be removed.